.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -43,17 +43,47 @@ If you need to submit a pr for any of these repos during this period, you should
 
 - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
```

Comment:
Agree, we shouldn't, the only reason I haven't removed it is because this is the first time through, so lets just be careful first time, in case we have to fall back to building with master. I shall add a comment to that effect here.